.generator/src/generator/cli.py

```diff
@@ -77,15 +73,17 @@ def cli(input, output):
     apis = openapi.apis(spec)
     models = openapi.models(spec)
 
-    package = output / config["packageName"].replace(".", "/")
-    package.mkdir(parents=True, exist_ok=True)
+    top_package = output / "datadog_api_client"
```

Comment:
could we have this constant somewhere defined?

Reply:
Why verbose? We can simply set the values in Jinja environment without passing the config itself.